index.html

```diff
@@ -1,28 +1,49 @@
 <!DOCTYPE html>
```

Comment:
Here is my banner for winning:

    <div ng-show="win" ng-class="{'has-success': win}">
        Congrats! You Won!
    </div>